test.properties

```diff
@@ -6210,6 +6210,7 @@
         ${project.dir}/modules/apps/depot/depot-test/src/testFunctional,\
         ${project.dir}/modules/apps/digital-signature/digital-signature-test/src/testFunctional,\
         ${project.dir}/modules/apps/document-library-opener/document-library-opener-google-drive-test/src/testFunctional,\
+        ${project.dir}/modules/apps/frontend-css/frontend-css-test/src/testFunctional,\
```

Comment:
I think it's because the test files were relocated on the last PR, so I had to add this to test.properties to run the tests locally. But I'll keep watch for both tests on upstream :+1:

Reply:
ah that makes sense, thanks!